packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/CollectingLocality/CollectingPosition.js

```diff
@@ -0,0 +1,189 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { pathBuilder } from 'coreModules/form/higherOrderComponents'
+import { Icon } from 'semantic-ui-react'
+
+const iconStyle = {
+  float: 'left',
+}
+
+const propTypes = {
```

Comment:
as a convention we put the propTypes and defaultProps closest to the component (like this: like this: https://github.com/airbnb/javascript/tree/master/react#ordering, although that example doesn't have other stuff), and if there's mapStateToProps/mapDispatchToProps we put them above the propTypes, and if there's other stuff we put it above them.

so the order in this case would be:

getCoordinates and similar
mapStateToProps
propTypes
defaultProps
Component...